map3d.py

```diff
@@ -0,0 +1,133 @@
+
+
+
+def map3d_surface(xdata,ydata,zdata,scalardata,mode,vmin,vmax,data_cmap='blue-red',data_alpha=1,topo=None,topo_limits=None,zscale=500.,topo_vmin=None,topo_vmax=None,topo_cmap='bone',topo_cmap_reverse=False,land_constant=False,land_color=(0.7,0.7,0.7),set_view=None):
+    """
+    xdata = longitude values for data array
+    ydata = latitude values for data array
+    zdata = depth values for data array
+    vmin = colorbar minimum for data
```

Comment:
missing comments for `scalardata` and `mode`

Reply:
done